mail_message_active/README.rst

```diff
@@ -0,0 +1,17 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+Mail message active
+===================
+
+* This module define "Active" fiend in "Message" object.
```

Comment:
This module defines...

Reply:
Realizadas las mejoras solicitadas por @agaldona , y vuelto a subir el módulo. Eskerrik asko.